src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -35,12 +39,163 @@
+            // -->
+            tagProcessor.registerTag(ListTag.class, "active_mobs", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (ActiveMob entity : getMobManager().getActiveMobs()) {
+                    list.addObject(new MythicMobsMobTag(entity));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.spawners>
+            // @returns ListTag(MythicSpawnerTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all MythicSpawners.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "spawners", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (MythicSpawner spawner : MythicMobsBridge.getSpawnerManager().getSpawners()) {
+                    list.addObject(new MythicSpawnerTag(spawner));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.damage_modifiers[<mob_id>]>
+            // @returns MapTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a map of the damage modifiers for a MythicMob from a Mob ID.
+            // -->
+            tagProcessor.registerTag(MapTag.class, ElementTag.class, "damage_modifiers", (attribute, object, id) -> {
+                Optional<MythicMob> optionalMythicMob = MythicBukkit.inst().getMobManager().getMythicMob(id.asString());
+                if (!optionalMythicMob.isPresent()) {
+                    return null;
+                }
+                MythicMob mob = optionalMythicMob.get();
+                MapTag result = new MapTag();
+                for (Map.Entry<String, Double> entry : mob.getDamageModifiers().entrySet()) {
+                    result.putObject(entry.getKey(), new ElementTag(entry.getValue()));
+                }
+                return result;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.mob_path[<mob_id>]>
+            // @returns ElementTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns the path to the configuration for the MythicMob from a Mob ID.
+            // -->
+            tagProcessor.registerTag(ElementTag.class, ElementTag.class, "mob_path", (attribute, object, id) -> {
+                MythicMob mob = getMythicMob(id.asString());
+                if (mob == null) return null;
+                return new ElementTag(mob.getConfig().getFile().getPath(), true);
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.item_path[<item_id>]>
+            // @returns ElementTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns the path to the configuration for the MythicItem from an Item ID.
```

Comment:
Documentation and/or tag name should have keyword `file` in it, as this appears to be a raw filepath.